stable/sematext-agent/values.yaml

```diff
@@ -0,0 +1,63 @@
+agent:
+  image:
+    repository: sematext/agent
+    tag: latest
+    pullPolicy: Always
+  service:
+    port: 80
+    type: ClusterIP
+  config:
+    JOURNAL_DIR: /opt/spm/st-agent
+    PIPELINE_CONSOLE_OUTPUT: false
+    PIPELINE_NULL_OUTPUT: false
+    API_SERVER_HOST: 0.0.0.0
+    LOGGING_WRITE_EVENTS: false
+    LOGGING_REQUEST_TRACKING: false
+    AUTODISCO_TEMPLATES_PATH: /etc/agent/autodisco.yml
+    LOGGING_LEVEL: warn
+  resources: {}
+
+logagent:
+  image:
+    repository: sematext/logagent
```

Comment:
Please provide a comment of where the source is for this image.

Reply:
Looks like I can trace it through DockerHub. I'm okay with that

Reply:
This is logagent repo: https://github.com/sematext/logagent-js. Maybe I should add it to readme.